fontir/src/ir.rs

```diff
@@ -85,6 +81,43 @@ impl StaticMetadata {
     pub fn glyph_id(&self, name: &GlyphName) -> Option<u32> {
         self.glyph_order.get_index_of(name).map(|i| i as u32)
     }
+
+    pub fn default_metrics(&self) -> &Metrics {
+        self.metrics
+            .get(self.variation_model.default_location())
+            .unwrap()
+    }
+
+    pub fn default_metrics_mut(&mut self) -> &mut Metrics {
+        self.metrics
+            .get_mut(self.variation_model.default_location())
+            .unwrap()
+    }
+}
+
+/// Global metrics. Ascender/descender, cap height, etc.
+///
+/// Represents the values of these metrics at a specific position in design space.
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq)]
+pub struct Metrics {
+    pub ascender: OrderedFloat<f32>,
+    pub descender: OrderedFloat<f32>,
+    pub cap_height: OrderedFloat<f32>,
+    pub x_height: OrderedFloat<f32>,
```

Comment:
technically MVAR allows each one of these (and similar other related ones) to variate independently of the others. Maybe we should support that somehow.
Either make these struct fields all optional such that when None we take it as "does not participate" when making the MVAR deltas, or we have map of {location: value} for each one of them, instead of a map {Location: Metric}

Reply:
This would also be in line with our statede intention to move away from a "multiple-master" approach to font variations

Reply:
Good point, I'll update to make them more independent.

Reply:
Done